packages/backend-data/API.md

```diff
@@ -63,6 +70,9 @@ export type LambdaAuthorizationModeProps = {
     timeToLiveInSeconds?: number;
 };
 
+// @public
+export type ModelTableNameMap = Record<string, string>;

```

Comment:
I would inline this above and delete this type. It doesn't seem to bring much value and will show up in intellisense